### DISCUSSION TITLE
[SC]: R012 firma dichairante

### DISCUSSION THREAD

**Discussion by OrazioCostanzo** [italia/ansc#876] [Q&A] [answered]
### Ambiente

pre-produzione

### Servizio

R012 - Servizi cooperativi firma elettronica del dichiarante

### idOperazione

assente

### Provenienza segnalazione

Software House

### Comune di riferimento e operatore

COMUNETEST052032 - INF.OR

### Descrivi il problema

Salve,
vorrei sapere per ogni tipo di atto (Cittadinanza, Morte, Nascita, Matrimonio, Unione civile) quale figura, il servizio,  accetta come firmatario e quindi idoneo per la ricezione del link per la firma.
Grazie.

**Comment by amusarra** [comment, marked as answer]
Salve @OrazioCostanzo 
Vale per qualunque tipo evento, basta chiamare il servizio preposto con il template di payload a seguire. Il destinatario deve essere opportunamente **"fleggato"** e in particolare `flagFirmatario` pari a `true`.

```
{
	"testataRichiesta": {
		"idComune": ${id_comune_operatore},
		"idOperazioneComune": "${__threadGroupName}_${__threadNum}",
		"dataOraRichiesta": ${__time("yyyy-MM-dd'T'HH:mm:ss.SSSXXX",)},
		"nomeApplicativo": ${nome_applicativo},
		"versioneApplicativo": ${versione_applicativo},
		"fornitoreApplicativo": ${fornitore_applicativo}
	},
    "emailDichiarante": "${email_firmatario_p}",
    "idEvento": ${id_evento},
    "idAnsc": "${id_ansc}",
    "firmatario": {
        "nome": "${nome_p}",
        "cognome": "${cognome_p}"
    }
}
```
Payload 1 - Esempio di template per il servizio di presavisione.

Converto la issue in discussion in quanto non si tratta di un bug.

Cordiali Saluti.
Supporto ANSC.

**Comment by OrazioCostanzo** [comment]
Per un soggetto che compare nell'atto, basta solo il flagFirmatario pari a true?

**Reply by mttfranci** [reply]
Si, esattamente.
